### ISSUE TITLE
Is it support to distrubuted

### ISSUE THREAD

**Issue by masa2146** [socketio/engine.io-server-java#40]
Hello, I want to create socket communication using with distributed system. I found this [netty-socketio](https://github.com/mrniko/netty-socketio) project bu it's not active in development.  does this project support distrubuted structure or will it?

**Comment by trinopoty** [comment]
I think you're looking for socket.io implementation. Refer to this repo: https://github.com/socketio/socket.io-server-java/
And, to answer you question, yes it's possible to have distributed. But, since I cannot know your specific architecture, you will need to implement such logic. You can use the `SocketIoMemoryAdapter` class as a reference.
Instances of `SocketIoSocket` are not distributed, distribution is implemented on top of rooms.
Basically, you need to implement a class that extends `SocketIoAdapter` which will be called by the library when it needs to send some packets to rooms.
